recipes/cif-tools/run_test.sh

```diff
@@ -3,8 +3,11 @@
 set -exu
 
 pdb2cif 7f95-carb.pdb
-cif2pdb 7f95-carb.cif
-cif-diff 443d_final.cif 7f95-carb.cif
+
+# buggy commands
```

Comment:
Buggy because of upstream? Is there an upstream issue?

Reply:
The issue of `mmcif_pdbx.dic` not being found and segmentation faults error occurring in `cif2pdb` and `cif-diff`, respectively.

I confirmed that `mmcif_pdbx.dic` exists when building locally.

Furthermore, since the commands worked correctly in previous versions, I suspect this may be an upstream issue.